### PR TITLE
Responsive-Grid/Portfolio/Mo

### PR DESCRIPTION
**Pull request by mnashawati**

### CODE REVIEW

**Comment by sophiedermaux** [comment]
Great job Mo!! You can see you experimented with the columns and that is great. I would recommend in just ensuring your indentation is clear and always in line for reviewing

File: css/main.css [hunk 1]
```diff
@@ -1,3 +1,200 @@
...
+    display: flex;
+    justify-content: center;
+}
+
+header ul li {
+    list-style-type: none;
+    padding: 1rem;
+}
+
+header ul li a {
```
Comment (theseafaringturtle) [inline css/main.css]: It's probably best to use a class for this for maintainability

File: css/main.css [hunk 2]
```diff
@@ -1,3 +1,200 @@
...
+header ul li a:hover {
+    opacity: 1;
+}
+
+h2 {
+    padding-top: 1rem;
+}
+
+.hero {
+    background-image: url('../imgs/photo-1490117874548-e35a2286fd89.jpeg');
```
Comment (theseafaringturtle) [inline css/main.css]: ../ is not necessary here since / starts from the site's root folder

File: index.html [hunk 3]
```diff
@@ -6,14 +6,100 @@
...
+
+        <section class="third-bg-img">
+        </section>
+
+        <section id="contact" class="contact">
+            <h1>Contact.</h1>
+            <div class="container">
+                <form class="row">
+                    <label class="col-10 sm-col-10 md-col-6 lg-col-5 info-label" for="name">Enter Your Name</label>
+                    <input class="col-10 sm-col-10 md-col-6 lg-col-5" type="text" id="name" name="name" value="Name" required><br>
```
Comment (theseafaringturtle) [inline index.html]: I recommend using placeholder= rather than value= in inputs

**Comment by mnashawati** [comment]
> Great job Mo!! You can see you experimented with the columns and that is great. I would recommend in just ensuring your indentation is clear and always in line for reviewing

Thank you for the feedback Sophie! Could you please explain what exactly I can do about the indentation? I use "Format Document with Beautify" for HTML in VS Code so it is automatic.

File: index.html (hunk 3 above)
Comment (mnashawati) [reply index.html]: Thank you for the note Alessandro :) Noted and done!

**Pull request closed without merging**